doc/changes/0.2.0.rst

```diff
@@ -1,8 +1,23 @@
 Changelog for Falcon 0.2.0
 ==========================
 
-New
----
+Breaking Changes
```

Comment:
Just reordered the sections to be consistent with the 1.0.0 release notes. I wanted to make sure people didn't gloss over the breaking changes and miss something that affects them.